samples/spring-boot-hello-world/pom.xml

```diff
@@ -66,17 +66,8 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-war-plugin</artifactId>
-                <version>3.3.2</version>
```

Comment:
I suggested to remove just version number keeping everithing else without changes. Does it work actually?